scripts/js/queries.js

```diff
@@ -245,30 +245,41 @@ function formatReplyTime(replyTime, type) {
   return replyTime;
 }
 
-function formatInfo(data) {
-  // DNSSEC status
-  var dnssecStatus = data.dnssec,
-    dnssecClass;
-  switch (data.dnssec) {
+// Parse DNSSEC status
+function parseDNSSEC(data) {
+  var icon = null, // Icon to display
+    color = false, // Class to apply to text
+    text = data.dnssec; // Text to display
+  switch (text) {
     case "SECURE":
-      dnssecClass = "text-green";
+      icon = "fa-solid fa-lock";
```

Comment:
I was looking at your screenshot and I have a suggestion.

It would be easier to read the table if the icons were aligned:

We can add `pull-right` class to every icon:
 ```suggestion
      icon = "fa-solid fa-lock pull-right";
```
![image](https://github.com/user-attachments/assets/65e5e335-6c76-49b2-a68b-aa9c80bd6f67)

Or 

Move the icon to the beginning of the text (change line 609 to):
```
    domain = dnssecIcon + " " + utils.escapeHtml(domain);
```
![image](https://github.com/user-attachments/assets/6460d0c3-ede3-4246-9a9f-a29422db4577)

What do you think?

Reply:
I like the icon on the left side of the domain name. But I would increase the space between icon and text a bit more (or even a separate column?)

Reply:
A separate column will take too much space.
We can add an extra non-breaking space (`&nbsp;`) to increase the space:
```
domain = dnssecIcon + "&nbsp; " + utils.escapeHtml(domain);
```

Reply:
@rdwebdesign Does an "empty icon" exist? I'm thinking about the alignment of `DS` records which do not receive an icon.

Reply:
We could also move the icon to the other status icon... Just suggesting this here, not overly convinced right now.

![image](https://github.com/user-attachments/assets/78640ad1-c7e9-495c-b4b2-4c3c10e32c48)

or

![image](https://github.com/user-attachments/assets/6e4fb4e5-e83e-4b45-aefa-58a8f9802a76)

Reply:
I think this is to much icon at the same place

Reply:
> Does an "empty icon" exist? I'm thinking about the alignment of DS records

I don't think there is an empty icon, but we can put all icons (including the "no icon") inside a `<span>` tag with a fixed width

Reply:
Please try the changes from my last commit.

Feel free to change or remove the commit if you think there is a better way to do it.

Reply:
With the latest commit I get 
![2024-12-21_22-01](https://github.com/user-attachments/assets/0bf45b72-8842-4d44-9f39-fbdd5fbe5fb9)

I think this was not the intention...

Reply:
Oh... I tested directly on the container and forgot to copy one of the changes to the repo.

I'll fix it in a few minutes.

Reply:
Done.

Reply:
Looks nice.
![2024-12-21_22-43](https://github.com/user-attachments/assets/5ed0a0c8-a516-4df4-a679-1b9b64cf0eec)